File: app/src/main/java/ru/rewindforce/concerts/HomeScreen/Concert.java
package ru.rewindforce.concerts.HomeScreen;

import java.io.Serializable;

public class Concert implements Serializable {

    private int concert_id, crowd, bands_count;
    private String title, club, description;
    private long datetime;

    public int getId() { return concert_id; }
    public String getBand() { return title; }
    public String getClub() { return club; }
    public int getCrowd() { return crowd; }
    public long getDatetime() { return datetime; }
    public int getBandsCount() { return bands_count; }
    public String getDescription() { return description; }
}

File: app/src/main/java/ru/rewindforce/concerts/data/Concert.kt
package ru.rewindforce.concerts.data

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

private const val HOST: String = "https://rewindconcerts.000webhostapp.com"

@Parcelize
data class Concert(val UID: String = "", val title: String = "", val club: String = "",
                   val description: String = "", val lowresPoster: String = "",
                   val highresPoster: String = "", val city: String = "",
                   val lineUp: ArrayList<String> = ArrayList(),
                   val datetime: Long = 0,
                   var commentsCount: Int = 0) : Parcelable {
    val lowresPosterURL: String
        get() = if (lowresPoster == "/photoalbums/defaultheader.jpg") HOST +lowresPoster
                else "$HOST/concertposters/$UID/lowres/$lowresPoster"

    val highresPosterURL: String
        get() = if (highresPoster == "/photoalbums/defaultheader.jpg") HOST +highresPoster
                else "$HOST/concertposters/$UID/highres/$highresPoster"

    val bandsCount: Int
        get() = lineUp.size

    fun incrementCommentsCount() = commentsCount++
    fun decrementCommentsCount() = commentsCount--
}